app/views/pages/privacy_policy.html.erb

```diff
@@ -0,0 +1,8 @@
+<!-- Is this rendering correctly? -->
+<div class="content__left">
+  <%= link_to 'Back', 'javascript:history.go(-1);', class: "govuk-back-link hidden", id: "backlink" %>
```

Comment:
Does this not need to work without JS enabled?

Reply:
Yes it is default hidden, the <%= javascript_pack_tag 'back_link'%> displays it

Reply:
I think without JS enabled the pattern is it should not be visible, or  be visible and go back to the previous page, for which I need to implement a previous step method for all models now that the flow is locked down. WIP...